packages/mobile/src/exchange/saga.ts

```diff
@@ -258,7 +261,9 @@ export function* exchangeGoldAndStableTokens(action: ExchangeTokensAction) {
         convertedMakerAmount.toString()
       )
     } else {
-      ValoraAnalytics.track(AnalyticsEvents.unexpected_maker_token, { makerToken })
+      ValoraAnalytics.track(CeloExchangeEvents.celo_exchange_error, {
+        error: `Unexpected maker token: ${makerToken}`,
+      })
```

Comment:
Perhaps outside the scope of this PR but in Exchange, we see transactions fail if a user only has Celo, as the wallet pays fees in cUSD only. Would be helpful to track when people attempt an exchange when they only have celo

Reply:
What net new information would we need to log for that? Wouldn't `makerToken` be `CELO` in that instance?

Reply:
We would want to log the celo balance in the case of an error- sorry this comment is separate from the line I commented on